doc/src/odin/dev_guide/fix.md

```diff
@@ -1,19 +1,32 @@
+Pull requests are intended to correct bugs and improve Odin's performance.
+To create a pull request, clone the vtr-verilog-rooting repository and branch from the master.
+Make changes to the branch that improve Odin II and correct the bug.
+**Important** In addition to correcting the bug, it is required that test cases (benchmarks) are created that reproduce the issue and are included in the regression tests.
+An example of a good test case could be the benchmark found in the "Issue" being addressed.
+The results of these new tests need to be regenerate. See [regression tests](./regression_tests) for further instruction.
+Push these changes to the cloned repository and create the pull request.
+Add a description of the changes made and reference the "issue" that it corrects. There is a template provided on GitHub.
 
 ### Creating a "Work in progress" (WIP) PR
 
 **Important** Before creating a WIP PR, if it is a bug you have happened upon and intend to fix make sure you create an issue beforhand.
 
-A "work in progress" PR is a pull request that isn't complete or ready to be merged. It is intended to demonstrate that a said Issue is being addressed and indicates to others that they don't need to fix it. To create a WIP PR, follow the same steps as making a PR. However, make a couple changes to the branch and then create a pull request with that branch, **INCLUDING** WIP in the title. This will automatically indicate that this PR is not ready to be merged. Like a regular PR, test cases are also needed to be included through the use of benchmarks. See [regression tests](./regression_tests) for further instruction.
+A "work in progress" PR is a pull request that isn't complete or ready to be merged.
+It is intended to demonstrate that an Issue is being addressed and indicates to others that they don't need to fix it.
+To create a WIP PR, follow the same steps as making a PR.
+However, make a couple changes to the branch, create a pull request with that branch, and **include WIP in the title.**
+This will automatically indicate that this PR is not ready to be merged.
+Like a regular PR, test cases are also needed to be included through the use of benchmarks.
+See [regression tests](./regression_tests) for further instruction.
 
 ### Formating
 
-Odin II uses clang formatting as it's standard. PRs will not be accepted if it does not adhere to the clang standard.
\ No newline at end of file
+Odin II uses clang formatting as it's standard. PRs will not be accepted if it does not adhere to the clang standard.
```

Comment:
actually can you change this to:

Odin II shares the same contributing philosophy as VPR (https://docs.verilogtorouting.org/en/latest/dev/contributing/contributing/), most importantly PR will be rejected if they do not respect the coding standard: see VPRs coding standard https://docs.verilogtorouting.org/en/latest/dev/developing/#code-formatting